### ISSUE TITLE
Why finetuning MVSNet

### ISSUE THREAD

**Issue by ldz666666** [apchenstu/mvsnerf#75]
Hello, thanks for your excellent work!
In the original paper, it seems that you have claimed that during finetuning, MVSNerF is free from the 2d feature extractor and the 3d cnn (Correpsonding to the MVSNet class of the code), which means the MVSNet doesn't need to be optimized during fintuning.

In introduction
> In essence, the encoding volume is a localized neural representation of the radiance field; once estimated, this volume can be used directly (dropping the 3D CNN) for final rendering by differentiable ray marching.

In section 3.4
> Note that, we optimize only the encoding volume and the MLP, instead of our entire network

However, in` train_mvs_nerf_finetuning_pl.py` and `models.py`.  It seems that the MVSNet also needs to be optimized during finetuning, which confuses me.

Could you please explain the difference of the paper and the code? Many thanks!

**Comment by apchenstu** [comment]
Thanks, we don't finetune the mvsnet, you may want to note that the [image_feature](https://github.com/apchenstu/mvsnerf/blob/1fdf6487389d0872dade614b3cea61f7b099406e/renderer.py#L138) is none during finetuning, so the gradient wouldn't pass through the mvsnet.

**Comment by basit-7** [comment]
@apchenstu **img_feat** are set to none in the training script as well, why is that? https://github.com/apchenstu/mvsnerf/blob/1fdf6487389d0872dade614b3cea61f7b099406e/train_mvs_nerf_pl.py#L123
